code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -71,11 +115,20 @@
 
 /obj/machinery/computer/camera_advanced/xenobio/attackby(obj/item/O, mob/user, params)
 	if(istype(O, /obj/item/reagent_containers/food/snacks/monkeycube))
-		monkeys++
-		to_chat(user, "<span class='notice'>You feed [O] to [src]. It now has [monkeys] monkey cubes stored.</span>")
-		user.drop_item()
-		qdel(O)
-		return
+		if(user.drop_item())
+			monkeys++
+			to_chat(user, "<span class='notice'>You feed [O] to [src]. It now has [monkeys] monkey cubes stored.</span>")
+			qdel(O)
+			return
+	else if (istype(O, /obj/item/slimepotion/slime))
+		var/replaced = FALSE
+		if(user.drop_item())
```

Comment:
```
if(user.drop_item())
	return
```

Reply:
Uh if I return immediately there, when do I fire the code I currently have inside of that `if(user.drop_item())`?

Reply:
Yeah, what you had is fine. Had a derp moment.